app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt

```diff
@@ -7,21 +7,28 @@ import io.github.droidkaigi.confsched2018.model.Date
 import io.github.droidkaigi.confsched2018.model.toReadableDateTimeString
 import io.github.droidkaigi.confsched2018.model.toReadableTimeString
 
-@BindingAdapter(value = ["bind:startDate", "bind:endDate", "bind:postFix"])
-fun TextView.setPeriodText(startDate: Date?, endDate: Date?, postFix: String) {
+@BindingAdapter(value = ["bind:startDate", "bind:endDate"])
+fun TextView.setPeriodText(startDate: Date?, endDate: Date?) {
     startDate ?: return
     endDate ?: return
     text = context.getString(
             R.string.time_period,
             startDate.toReadableTimeString(),
-            endDate.toReadableTimeString(),
-            postFix
+            endDate.toReadableTimeString()
     )
 }
 
-@BindingAdapter(value = ["bind:startDate", "bind:endDate"])
-fun TextView.setPeriodText(startDate: Date?, endDate: Date?) {
-    setPeriodText(startDate, endDate, "")
+@BindingAdapter(value = ["bind:prefix", "bind:roomName"])
+fun TextView.setRoomText(prefix: String?, roomName: String?) {
+    prefix ?: return
+    text = when(roomName) {
+        null -> ""
+        else -> context.getString(
```

Comment:
Probably this is short, So can you write by one line?

Reply:
Okay 🙌  https://github.com/DroidKaigi/conference-app-2018/pull/89/commits/9b60f21275d7b6beb719d3a91c9cd6a8ecc883e5 Is this right or are there any more short-handed code? 🙏